km_api/account/serializers.py

```diff
@@ -244,6 +244,12 @@ def validate_key(self, key):
             raise serializers.ValidationError(
                 _('This key is invalid.'))
 
+        if models.EmailAddress.objects.filter(
+                email=confirmation.email.email,
+                verified=True).exists():
+            raise serializers.ValidationError(
+                _('That email address has already been used.'))
```

Comment:
This piece looks good and seems to address the issue of multiple users verifying the same email address.